src/fullpage/fullpage.tex

```diff
@@ -34,8 +34,9 @@
 \usepackage{amsfonts,amsmath}
 
 \usepackage{fontspec}
-\newfontfamily\comicsans{Comic Sans MS}
-\setsansfont{Helvetica}
+% Using Linux-compatible fonts
+\newfontfamily\comicsans{DejaVu Sans}  % Alternative to Comic Sans MS
+\setsansfont{Liberation Sans}          % Alternative to Helvetica
```

Comment:
Ahh thank you!

Reply:
I'm not sure how will this work on macos or limbows. So I thought I could add a dockerfile with all the dependencies installed as an execution environment.

Reply:
I'd say docker is a big heavy, but we could make switching things easier.  I've forgotten if I'd trouble here on macos or linux, but most likely `\setsansfont{Helvetica}` breaks on macos too.